blueocean-pipeline-api-impl/src/test/java/io/jenkins/blueocean/rest/impl/pipeline/analytics/PipelinePluginAnalyticsTest.java

```diff
@@ -33,6 +33,7 @@ public void testGenerateAnalyticsEventWithDeclarative() throws Exception {
         Assert.assertEquals("timesUsed", 2, properties.get("timesUsed"));
         Assert.assertEquals("isDeclarative", true, properties.get("isDeclarative"));
         Assert.assertEquals("runResult", "SUCCESS", properties.get("runResult"));
+//        Assert.assertEquals("plugin", "", properties.get("plugin"));
```

Comment:
so it turns out `pluginManager.whichPlugin` doesn't work inside of JenkinsRule tests but does work when you run the app normally. Go figure :(

Reply:
Any ideas @vivek or @jglick ?

Reply:
@Jesse claimed to add something to the ArgumentsActionImpl that will give you a more direct way to get at this and may not need the plugin manager -- I haven't tried it out though.